polybar-scripts/info-optimus-manager/README.md

```diff
@@ -0,0 +1,23 @@
+# Script: info-optimus-manager
```

Comment:
I think we should rename this script and the directories to `system-gpu-optimus`.